tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -2252,13 +2252,13 @@ describe('Filing History List - with documents', () => {
     submitter: 'Cameron'
   }
 
-  it.skip('does not display the documents list when no documents are present on a filing', async () => {
+  it('does not display the documents list when no documents are present on a filing', async () => {
     // init store
     filingHistoryListStore.setFilings([FILING_WITH_DOCUMENTS_LINK] as any)
+    filingHistoryListStore.setPanel(null) // so we can re-open it
```

Comment:
This is the bug fix.

In this test suite (file), it's the same FHL Store, so unit tests can affect each other. To force the first panel to re-open, this state variable needs to be cleared.